bwrap.xml

```diff
@@ -130,6 +130,10 @@
       <term><option>--unshare-all</option></term>
       <listitem><para>Unshare all possible namespaces. Currently equivalent with: <option>--unshare-user-try</option> <option>--unshare-ipc</option> <option>--unshare-pid</option> <option>--unshare-net</option> <option>--unshare-uts</option> <option>--unshare-cgroup-try</option></para></listitem>
     </varlistentry>
+    <varlistentry>
+      <term><option>--share-net</option></term>
+      <listitem><para>Retain the network namespace (can only combine with <option>--unshare-all</option>)</para></listitem>
```

Comment:
What do you mean by "can only combine with" here? Obviously you can combine `--share-net` with `--bind`, for example.

Perhaps something like

```suggestion
      <listitem><para>Retain the network namespace, overriding an earlier <option>--unshare-all</option> or <option>--unshare-net</option></para></listitem>
```

would be clearer?

Reply:
I just copied the --help description of `--share-net` but you're right that this is a bit misleading.